src/utils/conf/base.config

```diff
@@ -12,6 +12,7 @@ params {
         file_converter {
             off = 'h5ad'
             tagCellWithSampleId = true
+            remove10xGEMWell = false
```

Comment:
This is backwards compatible in that it doesn't require the `remove10xGEMWell` key to be present in params. But, it's maybe important to note that the default behavior has changed (to now keep the suffix).

Reply:
Yeah this is important I think

Reply:
I think this can be stated in the release notes